app/styles/_icons.scss

```diff
@@ -252,10 +252,15 @@ i.fa-circle-o-notch.fa-spin {
 }
 
 .icon--search {
-  width: 21px;
-  height: 21px;
+  width: 20px;
+  height: 23px;  
 }
 
+.searchbox--slim .icon--search {
+  width: 14px;
+  height: 15px;
+} 
+
```

Comment:
is `.icon--search` used anywhere else? if not, you might be able to get away with just updating the base class.